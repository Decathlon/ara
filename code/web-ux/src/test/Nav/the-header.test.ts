import { mount } from '@vue/test-utils'
import { describe, expect, test } from 'vitest'
import Header from '../../components/Nav/TheHeader.vue'

describe('Header.vue', () => {
  test('Should open sidebar', async () => {
    const wrapper = mount(Header)

    await wrapper.find('button').trigger('click')

    wrapper.vm.$nextTick(() => {
      expect(wrapper.emitted()).toHaveProperty('opened-menu');
      expect(wrapper.text()).toBeTruthy();
    })
  })  
})