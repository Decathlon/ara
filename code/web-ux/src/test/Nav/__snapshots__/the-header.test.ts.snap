// Vitest Snapshot v1

exports[`suite name > snapshot 1`] = `
{
  "foo": "bar",
}
`;
